### ISSUE TITLE
Broken links on website

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#164]
```
Very minor issue...the links in the footer of the "Samples Studio" do not work 
as the page is hosted on a new domain and the links are relative, e.g. 
http://studio.html5rocks.com/tos.html. 
```

Original issue reported on code.google.com by `ian.tip...@gmail.com` on 17 Aug 2010 at 8:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `erne...@google.com` on 17 Aug 2010 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a8ff7edb2c.
```

Original comment by `paulir...@google.com` on 26 Aug 2010 at 12:10
- Changed state: **Fixed**
